.commitlintrc.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
       [
         'api',             // user-facing API (e.g. XRefService)
         'dev',             // developer tooling/scripting (e.g. linting)
-        'example',         // example tools or docs (e.g. sample web ui)
+        'example',         // example tools or docs (e.g. sample cross-reference service)
```

Comment:
The lint failure is because it wants to format this file because this line is too long. I think just replace "cross-reference" with "xref" and you'll be fine.